src/cryptoadvance/specter/templates/base.jinja

```diff
@@ -124,29 +124,12 @@
 				showNotification(`{{ _("Revealing") if specter.hide_sensitive_info else _("Hiding") }} sensitive info...`, 0);
 				let url = `{{ url_for('auth_endpoint.toggle_hide_sensitive_info') }}`;
 				var formData = new FormData();
-				formData.append('csrf_token', '{{ csrf_token() }}');
-				try {
-					const response = await fetch(
-						url,
-						{
-							method: 'POST',
-							body: formData
-						}
-					);
-					if(response.status != 200){
-						showError(await response.text());
-						return;
-					}
-					const jsonResponse = await response.json();
-					if (jsonResponse.success) {
-						location.reload();
-						return
-					};
-					showError(`{{ _("Failed to toggle sensitive info mode...") }}`)
-				} catch(e) {
-					console.log("Caught error: ", e);
-					showError(e);
-				}
+				
+				result = await send_request(url, 'POST', "{{ csrf_token() }}", formData);
```

Comment:
Seems to be general thing: It doesn't seem that the result / response vars have been declared before

Reply:
Fixed in https://github.com/cryptoadvance/specter-desktop/pull/2014/commits/f354b0b10e1e4c817dce3e044f9e3c5a815f0dc3